src/components/general/Speakers.vue

```diff
@@ -1,32 +1,172 @@
 <template>
 <div id="speakers">
-	<h2>Take a Look at CUTC 2020</h2>
-	<router-link v-if="limit" to="/speakers">View all speakers ></router-link>
+	<div class="header">
+		<h2>Take a Look at CUTC 2020</h2>
+		<router-link v-if="limit" to="/speakers" class="view-more">View all speakers ></router-link>
+	</div>
+
+	<div class="grid" ref="grid">
+		<div class="tile" v-for="(speaker, i) in speakers" :key="speaker">
+			<!-- PHOTO -->
+			<div class="image" :style="{'display': showTile(i)}">
```

Comment:
just for my own learning could you explain what `:style` does?

Reply:
Style is like css classes, but makes it so you don't have to create a class. The colon makes it dynamic (not just a string)